source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -407,3 +408,32 @@ function set_site_breadcrumbs() {
 
 	return $breadcrumbs;
 }
+
+/**
+ * Filters content for the code reference blocks so Table of Contents can be added.
+ *
+ * @param string $content
+ * @return string
+ */
+function filter_code_content( $content ) {
+	$post = get_post();
+
+	if ( ! is_single() && ! is_parsed_post_type( $post->post_type ) ) {
+		return $content;
+	}
+
+	return do_blocks(
+		'
+		<!-- wp:wporg/code-reference-summary /-->
+		<!-- wp:wporg/code-reference-description /-->
```

Comment:
I noticed while testing https://github.com/WordPress/wporg-mu-plugins/pull/332 that this is causing a fatal error — since [`the_content` is used in this block](https://github.com/WordPress/wporg-developer/blob/aa3dc20cc962e50ced78cbca85462f4eb64987f9/source/wp-content/themes/wporg-developer-2023/inc/template-tags.php#L1728), you get stuck in an infinite loop trying to render it.